src/core/services/layout/layout.spec.js

```diff
@@ -50,6 +52,13 @@ describe('layout directives', function() {
         var element = $compile('<div ' + attribute + '>Layout</div>')($rootScope.$new());
         expect(element.hasClass(expectedClass)).toBe(true);
       }));
+
+      it('should not add the class ' + expectedClass + ' if the body class has "md-css-only" ' + attribute, inject(function($compile, $rootScope, $document) {
```

Comment:
This was submitted in by a contributor and is pretty ingenious way to iterate values and create tests from arrays.